### ISSUE TITLE
Need any help?

### ISSUE THREAD

**Issue by KevinWardProf** [lmnitzsche/OpenWorldTextBasedGame#1]
@lmnitzsche Just wondering if there is anything you need/want for this game. Interested in collaborating on something.

**Comment by lmnitzsche** [comment]
Yeah, it's a huge project, so I'm open to collaboration!  If you want to help, create some mini-games in c++ that would be good for the game and if they're good (and you give me permission to use your code), I'll include them in the game and credit you!

**Comment by KevinWardProf** [comment]
@lmnitzsche 

That's sound great. I don't happen to see your coding standards listed anywhere on the repo. Additionally, I think it would be great to also discuss at more length what your expectations are for these mini-games. So long as my name is not stricken from the record and we share credit in our collaboration I give you license to use my code in whatever capacity you desire.
